### PR TITLE
samples: nrf9160: http_update: Update mfw

### PR DESCRIPTION
**Pull request by richardmccrae-nordicsemi**
Update modem firmware used in samples to 1.3.3 and 1.3.4

### CODE REVIEW

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -275,6 +275,14 @@ Bluetooth mesh samples
 nRF9160 samples
 ---------------
 
+* :ref:`http_full_modem_update_sample` sample:
+
+  * Use modem firmwares 1.3.3 and 1.3.4
+
+* :ref:`http_modem_delta_update_sample` sample:
+
+  * Use modem delta images for 1.3.4 to and from 1.3.4-FOTA-TEST
```
Comment (divipillai) [inline doc/nrf/releases/release-notes-changelog.rst:284]:
```suggestion
  * The sample now uses modem firmware v1.3.4 to do a delta update.
```
Please confirm whether rephrased sentence is correct?

**Comment by rlubos** [comment]
@richardmccrae-nordicsemi Please squash the doc review commits (and rebase).

**Review by divipillai — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Comment by rlubos** [comment]
@richardmccrae-nordicsemi Please check the compliance result, the co-authored entry seems wrong.

**Comment by richardmccrae-nordicsemi** [comment]
@rlubos The CI/Jenkins/test-ci-nrfconnect-boot-fw-update should not be considered as this downstream job is buggy at the moment. @ChristopherBittner to confirm.

**Pull request merged**